lib/github-checks.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+const chalk = require('chalk');
+const execa = require('execa');
+const ghGot = require('gh-got');
+const hostedGitInfo = require('hosted-git-info');
+const pTimeout = require('p-timeout');
+const pWaitFor = require('p-wait-for');
+const terminalLink = require('terminal-link');
+const util = require('./util');
+
+const THIRTY_SECONDS = 1000 * 30;
+const THIRTY_MINUTES = 1000 * 60 * 30;
+
+module.exports = async task => {
+	const hasUnpushedCommits = await execa.stdout('git', ['cherry']) !== '';
+
+	if (hasUnpushedCommits) {
+		await execa('git', ['push']);
+	}
+
+	const latestCommit = execa.stdout('git', ['rev-parse', 'HEAD']);
+	const {title} = task;
+	const pkg = util.readPkg();
```

Comment:
`util.readPkg();` call is used in many places of the codebase, which is wasteful as we only need to read the package twice in total. One before publish and once after. One be good to at least move the first case to the util file.

Reply:
With f54d8ee56d9af490acb8b05f9ce11a2e4756683e, I’m now passing `pkg` from `index.js` instead of calling `util.readPkg()` inside `github-checks.js`.